mappings/net/minecraft/entity/decoration/ItemFrameEntity.mapping

```diff
@@ -2,6 +2,7 @@ CLASS net/minecraft/unmapped/C_hywnolpn net/minecraft/entity/decoration/ItemFram
 	FIELD f_ccszafxh ITEM_STACK Lnet/minecraft/unmapped/C_rinmcaxy;
 	FIELD f_dnmfuadb itemDropChance F
 	FIELD f_hkngbiyl ROTATION Lnet/minecraft/unmapped/C_rinmcaxy;
+	FIELD f_rdvivgoi NUM_ROTATIONS I
```

Comment:
```suggestion
	FIELD f_rdvivgoi MAX_ROTATIONS I
```
maybe? depending on what this value actually is, anyway

Reply:
I suspect this refers to the rotation of the item in the frame which can be changed by right clicking and read by a comparator. 

If this is the case, I think `NUM_ROTATIONS` is appropriate. I would describe it as "The number of different rotations an item in an item frame can take."

`MAX_ROTATIONS` would be the number of times you can rotate it, which isn't limited.

Reply:
```suggestion
	FIELD f_rdvivgoi ROTATIONS_COUNT I
```